app/controllers/application_controller.rb

```diff
@@ -5,6 +5,31 @@ class ApplicationController < Sinatra::Base
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "password_security"
+  end
+
+  get '/' do
+    if logged_in?
+      @user = current_user
+      redirect '/tweets'
+    else
+      erb :index
+    end
+  end
+
+  helpers do
+    def logged_in?
+      !!session[:user_id]
+    end
+
+    def current_user
+      User.find(session[:user_id])
+    end
+    
+    def find_tweet(params)
```

Comment:
this would be better placed in the tweets controller, but it really only does one things that is already abstracted with an ActiveRecord method. I would probably not use this.